test/test_examples_3d_p4est.jl

```diff
@@ -0,0 +1,61 @@
+module TestExamples3dP4est
+
+using Test
+using Trixi
+
+include("test_trixi.jl")
+
+# pathof(Trixi) returns /path/to/Trixi/src/Trixi.jl, dirname gives the parent directory
+EXAMPLES_DIR = joinpath(pathof(Trixi) |> dirname |> dirname, "examples", "3d")
+
+@testset "P4estMesh" begin
+  @testset "elixir_advection_basic_p4est.jl" begin
```

Comment:
You should use `@trixi_testset` introduced in https://github.com/trixi-framework/Trixi.jl/pull/644.